docs/src/simulation_tutorial.md

```diff
@@ -0,0 +1,673 @@
+
+You can use the `parameparametricbootstrap()` function to run `nsims` iterations of data sampled using the parameters from `kb07_m`.
+Set up a random seed to make the simulation reproducible. You can use your favourite number.
+
+To use multithreading, you need to set the number of cores you want to use.
+E.g. in Visual Studio Code, open the settings (gear icon in the lower left corner or cmd-,) and search for "thread".
+Set `julia.NumThreads` to the number of cores you want to use (at least 1 less than your total number).
+
+
+Set random seed for reproducibility:
+```@example Main
+rng = StableRNG(42);
+```
+
+Run nsims iterations:
+```@example Main
+kb07_sim = parametricbootstrap(rng, nsims, kb07_m, use_threads = false);
+```
+**Try**: Run the code above with or without `use_threads = true`.
+
+The output DataFrame `kb07_sim` contains the results of the bootstrapping procedure. 
+
+```@example Main
+df = DataFrame(kb07_sim.allpars);
+first(df, 9)
+nrow(df)
+```
+
+The dataframe df has 9000 rows: 9 parameters, in 1000 iterations.
+
+TODO: NEED HELP: is there a facet_wrap function in Gadfly? 
+========
+
+Plot some bootstrapped parameter:
+```@example Main
+σres = @where(df, :type .== "σ", :group .== "residual").value
+plot(x = σres, Geom.density, Guide.xlabel("Parametric bootstrap estimates of σ"), Guide.ylabel("Density"))
+
+βInt = @where(df, :type .== "β", :names .== "(Intercept)").value
+plot(x = βInt, Geom.density, Guide.xlabel("Parametric bootstrap estimates of β (Intercept)"), Guide.ylabel("Density"))
+
+βSpeaker = @where(df, :type .== "β", :names .== "spkr: old").value
+plot(x = βSpeaker, Geom.density, Guide.xlabel("Parametric bootstrap estimates of β Speaker"), Guide.ylabel("Density"))
+
+βPrecedents = @where(df, :type .== "β", :names .== "prec: maintain").value
+plot(x = βPrecedents, Geom.density, Guide.xlabel("Parametric bootstrap estimates of β Precedents"), Guide.ylabel("Density"))
+
+βLoad = @where(df, :type .== "β", :names .== "load: yes").value
+plot(x = βLoad, Geom.density, Guide.xlabel("Parametric bootstrap estimates of β Load"), Guide.ylabel("Density"))
+```
+
+Convert p-values to dataframe and save it as CSV
+```@example Main
+kb07_sim_df = DataFrame(kb07_sim.coefpvalues);
+CSV.write("kb07_sim.csv", kb07_sim_df);
+```
+
+Have a look at your simluated data:
+```@example Main
+print(first(kb07_sim_df, 8))
+```
+
+Now that we have a bootstrapped data, we can start our power calculation.
+
+### Power calculation
+
+The function `power_table()` from `MixedModelsSim` takes the output of `parametricbootstrap()` and calculates the proportion of simulations where the p-value is less than alpha for each coefficient.
+You can set the `alpha` argument to change the default value of 0.05 (justify your alpha ;).
+
+```@example Main
+ptbl = power_table(kb07_sim)
+```
+
+TODO:  NEED HELP, alpha doesnt work anymore? Is it still possible to specify significance level?
```

Comment:
Ah, it changed from a keyword argument to a normal one (this distinction is stricter in Julia than in R), you should be able to do:

```
power_table(kb07, 0.075)
```

As actual users: does that seem weirder to than having to write explicitly `alpha=`?

Reply:
cool thanks. We fixed it. Daniel says it is more intuitive if arguments can be written explicit "alpha = ". I think its fine as long as its clear in the docs.